Formula/mesa.rb

```diff
@@ -4,6 +4,7 @@ class Mesa < Formula
   desc "Graphics Library"
   homepage "https://www.mesa3d.org/"
   license "MIT"
+  revision 1
```

Comment:
`Mako` is a build-only dependency. Unless `mesa` was hitting the bug that was fixed in `Mako` 1.2.4 (doesn't seem likely since bug had caused an error), a revision bump is probably unnecessary.